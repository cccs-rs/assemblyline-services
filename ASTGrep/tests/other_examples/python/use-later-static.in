token = "hahaha not this"
token = "correct value"

print(token)
