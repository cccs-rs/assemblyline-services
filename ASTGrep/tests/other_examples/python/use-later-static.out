token = "hahaha not this"
token = "correct value"

print("correct value")
